examples/bri-3/src/bri/zeroKnowledgeProof/services/circuit/circuitInputsParser/circuitInputParser.service.ts

```diff
@@ -0,0 +1,95 @@
+import { Injectable } from '@nestjs/common';
+import { LoggingService } from '../../../../../shared/logging/logging.service';
+
+@Injectable()
+export class CircuitInputsParserService {
+  constructor(private readonly logger: LoggingService) {}
+
+  public applyMappingToJSONPayload(payload: string, cim: CircuitInputsMapping) {
+    const result: any = {};
+
+    try {
+      const jsonPayload = JSON.parse(payload);
+
+      for (let mapping of cim.mapping) {
+        const value = this.getJsonValueByPath(
+          jsonPayload,
+          mapping.payloadJsonPath,
+        );
+
+        if (!value && !mapping.defaultValue) {
```

Comment:
should we check value === undefined, if value can be empty string? since we are checking ===undefined in method itself